utilities/inference_utils.py

```diff
@@ -118,11 +117,12 @@ def is_service_exposed(self) -> bool:
         """
         labels = self.inference_service.labels
 
-        if (
-            isinstance(self.inference_service, InferenceService)
-            and self.deployment_mode in KServeDeploymentType.RAW_DEPLOYMENT
-        ):
-            return labels and labels.get(Labels.Kserve.NETWORKING_KSERVE_IO) == Labels.Kserve.EXPOSED
+        if self.deployment_mode == KServeDeploymentType.RAW_DEPLOYMENT:
+            if isinstance(self.inference_service, InferenceGraph):
+                # For InferenceGraph, the logic is similar as in Serverless. Only the label is different.
+                return not (labels and labels.get(Labels.Kserve.NETWORKING_KSERVE_IO) == "cluster-local")
+            else:
+                return labels and labels.get(Labels.Kserve.NETWORKING_KSERVE_IO) == Labels.Kserve.EXPOSED
 
         if self.deployment_mode == KServeDeploymentType.SERVERLESS:
             if labels and labels.get("networking.knative.dev/visibility") == "cluster-local":
```

Comment:
why don't we replace this hardcoded label name to `Labels.Kserve.NETWORKING_KSERVE_IO`?